core/guard/contract.go

```diff
@@ -13,13 +12,51 @@ import (
 	guardPb "github.com/tron-us/go-btfs-common/protos/guard"
 )
 
+func NewFileStatus(session *storage.FileContracts, contracts []*guardPb.Contract, configuration *config.Config) (*guardPb.FileStoreStatus, error) {
+	guardPid, escrowPid, err := getGuardAndEscrowPid(configuration)
+	if err != nil {
+		return nil, err
+	}
+
+	fileStoreMeta := guardPb.FileStoreMeta{
+		RenterPid:        session.Renter.Pretty(),
+		FileHash:         session.FileHash.String(), //TODO need to check
+		FileSize:         10000,                     //TODO need to revise later
+		RentStart:        time.Time{},               //TODO need to revise later
```

Comment:
contract meta has this info

Reply:
and if some fileds are not mandatory to have, you can leave it actually.

Reply:
I know the shard have such information, but these information should belong to file not the shard, copy it back from shard looks weird . So I want to reserve such fields for future modification, while I am sure so far Guard did not have process to use them.

Reply:
re-think about it, and use contract[0].startTime and endtime as temp solution.